markdown.dtx

```diff
@@ -16978,7 +16981,8 @@ return M
 
 local input
 if input_filename then
-  local input_file = assert(io.open(input_filename, "r"))
+  local input_file = assert(io.open(input_filename, "r"),
+    [[could not open file "#1" for reading]])
```

Comment:
Here, we would need `"]] .. input_filename .. [["` instead of `"#1"`.

Reply:
Fixed.